### PR TITLE
Feat: Filter functionality Added

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi Folks,

I have just added Filter functionality to my Ecomm app
can you please give PR reviews .

deploy preview :- https://deploy-preview-4--verdant-liger-fb0dca.netlify.app/

Thanks

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import tshirtImage1 from "../../pages/assets/images/tshirt-img/Tshirt image1.jpg";
 import tshirtImage2 from "../../pages/assets/images/tshirt-img/Tshirt image2.jpg";
 import tshirtImage3 from "../../pages/assets/images/tshirt-img/Tshirt image3.jpg";
-import tshirtImage4 from "../../pages/assets/images/tshirt-img/Tshirt image4.jpg";
-import tshirtImage5 from "../../pages/assets/images/tshirt-img/Tshirt image5.jpg";
-import tshirtImage6 from "../../pages/assets/images/tshirt-img/Tshirt image6.jpg";
+// import tshirtImage4 from "../../pages/assets/images/tshirt-img/Tshirt image4.jpg";
```
Comment (irajbhinde) [inline src/backend/db/products.js:4]: You can remove the commented code

**Review by sohan9819 — APPROVED** [review]
Sort filter is not getting cleared when clear is clicked . Rest everything looks good .

File: src/components/filter/Filter.js [hunk 2]
```diff
@@ -11,77 +12,77 @@ export const Filter = () => {
...
-                type="range"
-                name="price"
-                id="price"
-                min="0"
-                max="500000"
-                value="50"
-              />
-            </div>
+          <div className="sidebar_box sidebar_slider">
+
```
Comment (irajbhinde) [inline src/components/filter/Filter.js:19]: You can use prettier on vscode to get rid of unwanted spacing & get the spacings right

**Review by irajbhinde — COMMENTED** [review]
Looks good, few things you can work upon when you get time 

1) Unchecking checkbox is not working 
2) Multiple checkboxes are not getting selected

**Pull request merged**